docs/specification.md

```diff
@@ -634,27 +630,33 @@ type VoteType = Upvote | Downvote
 data VoteParam = VoteParam
   { proposal_id :: ProposalId
   , vote_type :: VoteType
+  , vote_amount :: Natural
   }
 
 data Parameter proposalMetadata
-  = Vote VoteParam
+  = Vote [VoteParam]
 ```
 
 Parameter (in Michelson):
 
 ```
-(pair %vote
-  (nat %proposal_id)
-  (bool %vote_type)
+(list
+  (pair %vote
+    (nat %proposal_id)
+    (pair
+      (bool %vote_type)
+      (nat %vote_amount)
+    )
+  )
 )
 ```
 
-- Let `n` denote the amount of tokens required by the associated proposal to vote.
-- Sender MUST have enough unfrozen tokens (i. e. `≥ n`).
+- Sender MUST have unfrozen tokens equal to `vote_amount` (1 unfrozen token is needed for 1 vote).
```

Comment:
Sender must have `vote_amount` or more. It may sound confusing because one may think that the balance must be equal to `vote_amount`, but in fact it can be greater.

Reply:
I have updated this, and add other detail related to `set_voting_period` and `set_quorum_threshold` as well.